src/tests/dfio/daos_fio.c

```diff
@@ -24,7 +24,7 @@
 /*
  * Implementation of DAOS File System Fio Plugin
  */
-#include <string.h>
```

Comment:
im still not sure what the issue there is.
is fio re-defining all the str functions? if yes then i guess we can remove that. if not, then im not sure what is going on

Reply:
strsep() is defined in ./_build.external/fio/oslib/strsep.h
It's also defined in /usr/include/bits/string2.h which is included in string.h

Simply remove sting.h from daos_fio.c won't help since fio.h also includes sting.h:
In file included from /usr/include/string.h:633:0,
                 from _build.external/fio/fio.h:13,
                 from src/tests/dfio/daos_fio.c:39:
_build.external/fio/os/../oslib/strsep.h:4:7: error: expected identifier or ‘(’ before ‘__extension__’
 char *strsep(char **, const char *);

Seems it needs to be handled in fio code.

Reply:
Any suggestion on workaround for this in DAOS code?

Reply:
use this:
+        denv.AppendUnique(CPPPATH=[FIO_DIR])
+        denv.Append(CCFLAGS="-U_FORTIFY_SOURCE -D_FORTIFY_SOURCE=0 -O0")
+        denv.AppendUnique(LIBPATH=[Dir("../../client/dfs")])

in your Sconscript